src/main/java/be/glever/antplus/fec/EquipmentType.java

```diff
@@ -0,0 +1,49 @@
+package be.glever.antplus.fec;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+public enum EquipmentType {
+    Treadmill((byte) 19),
+    Elliptical((byte) 20),
+    Rower((byte) 22),
+    Climber((byte) 23),
+    NordicSkier((byte) 24),
+    StationaryBike((byte) 25);
+
+    private byte value;
+
+    EquipmentType(byte i) {
+        value = i;
+    }
+
+    public static Optional<EquipmentType> valueOf(int value) {
+        return Arrays.stream(values())
+                .filter(legNo -> legNo.value == value)
+                .findFirst();
+    }
+
+    public byte value() {
+        return value;
+    }
+
+    @Override
+    public String toString() {
+        switch (this) {
```

Comment:
I'm constantly falling over the toString() and switch{} implementation, deciding what to do.
Is the toString() even necessary for you? If not, I'd rather just remove it (KISS/YAGNI).

Reply:
Oh, `Enum#toString` already returns `this.name()`.
Then yeah, this is not needed.
But... it would be nice to have a more human readable name here and for `HeartRateDataSource`.

I'm going to need it here, to let the user choose which device type they want to search for.

Reply:
Actually... If I'm ever going to do internationalization, I'm not going to use this libary's strings at all. So I guess, we don't need them here.

Reply:
See #6